core-libs/setup/ssr/optimized-engine/optimized-ssr-engine.spec.ts

```diff
@@ -1173,8 +1288,72 @@ describe('OptimizedSsrEngine', () => {
         flush();
       }));
     });
-  });
 
+    it('should reuse the same render for subsequent requests for the same rendering key and first request should timeout', fakeAsync(() => {
```

Comment:
IMHO this description doesn't describe the essence of this test idea. What about:

> should reuse the same *errored* render for subsequent requests for the same rendering key, *even if the first request timeed out*